esp_encrypted_img/include/esp_efuse_utilities.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: 2025-2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
+
+#include "esp_efuse.h"
+#include "esp_hmac.h"
+
+/**
+ * @brief Check if the HMAC key is burnt in efuse.
+ *
+ * @param hmac_key_id[in] The HMAC key ID to check.
+ *
+ * @return
+ *      - true If the HMAC key is burnt.
+ *      - false If the HMAC key is not burnt.
+ */
+bool is_hmac_key_burnt_in_efuse(hmac_key_id_t hmac_key_id);
```

Comment:
Please use `esp_encrypted` prefix for consistency. Interim, this header could stay private, and not part of `include` dir?